### PR TITLE
Check if user has root privileges through sudo

### PR DESCRIPTION
**Pull request by to-bar**
- Refactor OS validation
- Disable become in preflight playbook
- Check if user has root privileges through sudo
- Update docs

### CODE REVIEW

File: core/src/epicli/data/common/defaults/configuration/shared-config.yml [hunk 1]
```diff
@@ -7,33 +7,5 @@ specification:
   download_directory: /tmp # directory where files and images will be stored just before installing/loading
   vault_location: '' # if empty "BUILD DIRECTORY/vault" will be used
   vault_tmp_file_location: SET_BY_AUTOMATION
```
Comment (seriva) [inline core/src/epicli/data/common/defaults/configuration/shared-config.yml:9]: We dont want to have supported OSs configurable anymore?

Comment (to-bar) [reply core/src/epicli/data/common/defaults/configuration/shared-config.yml:9]: Yes, I've changed this intentionally.

**Review by mkyc — APPROVED** [review]
LGTM

**Review by seriva — APPROVED** [review]

**Pull request merged**